product-accelerators/apim/components/org.wso2.healthcare.apim.backendauth/src/main/java/org/wso2/healthcare/apim/backendauth/BackendAuthenticator.java

```diff
@@ -56,14 +62,49 @@ public boolean mediate(MessageContext messageContext) {
             log.debug("Backend authenticator mediator is invoked.");
         }
 
-        BackendAuthConfig config = backendAuthConfig.get(configName);
-        String accessToken;
-        BackendAuthHandler currentAuthenticator;
+        BackendAuthConfig config;
+        if ("INLINE".equals(configType)){
```

Comment:
We can initialize the backendauth config at the deployment time? and the values

Reply:
configs are initialised at the startup and held in a Map, here we are getting the configured element from that Map.